nilearn/region_extraction/SimpleThresholdExtractor.py

```diff
@@ -0,0 +1,128 @@
+"""
+Voxel_ratio_extractor
+"""
+
+import nibabel
+import numpy as np
+
+from scipy.ndimage import label
+from scipy.stats import scoreatpercentile
+
+from nilearn import plotting
+from nilearn._utils import check_niimg
+from nilearn._utils.niimg_conversions import concat_niimgs
+from nilearn.image import iter_img, new_img_like
+
+
+def extract_seperate_regions(img, img_data, affine, min_size):
+    """ Extract the components which have largest
+        connections given a thresholded image of the
+        components
+
+        Parameters
+        ----------
+        img: numpy array of 3D Nifti like object
+            contains a thresholded image of the components
+
+        Returns
+        -------
+        the seperated or segmented regions extracted from
+        the input image which contains a set of connected
+        maps
+
+    """
+    all_regions_img = []
```

Comment:
By convention in nilearn, the "img" suffix refers to nibabel.SpatialImage objects. You use a list here. Please remove this suffix as it can be troublesome for other devs.